### DISCUSSION TITLE
Could you send me the data by e-mail?

### DISCUSSION THREAD

**Discussion by screen-noguchi** [DS3Lab/dataperf-vision-debugging#6] [Q&A]
I am considering participating in the competition.
However, I cannot access Google Drive from my company's network and cannot download the data.
Is it possible for you to send the data to my e-mail address?

**Comment by xzyaoi** [comment]
@screen-noguchi, sure! Could you send me an email and then I can reply to you? 
My email addr: xiaozhe.yao@inf.ethz.ch
